staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/aggregator/aggregator.go

```diff
@@ -268,6 +284,15 @@ func (s *specProxier) handleGroupVersion(w http.ResponseWriter, r *http.Request)
 
 // Register registers the OpenAPI V3 Discovery and GroupVersion handlers
 func (s *specProxier) register(handler common.PathHandlerByGroupVersion) {
-	handler.Handle("/openapi/v3", http.HandlerFunc(s.handleDiscovery))
+	handler.Handle("/openapi/v3", metrics.InstrumentHandlerFunc("GET",
+		/* group = */ "",
+		/* version = */ "",
+		/* resource = */ "",
+		/* subresource = */ "openapi/v3",
```

Comment:
You're leading with a slash for `metrics.MonitorRequest` which is not consistent with this particular usage.